frontends/web/src/routes/settings/electrum.jsx

```diff
@@ -69,23 +71,22 @@ class ElectrumServer extends Component {
             if (data.success) {
                 this.setState({ electrumCert: data.pemCert });
             } else {
-                alert(data.errorMessage);
+                // @ts-ignore
+                alerted(data.errorMessage);
```

Comment:
I guess using just `alert` doesn't work? Maybe use `alertUser` instead? `alerted` feels strange. And what does TypeScript complain about here?